pagopa/io-app
# feat: [IOCOM-853] Add footer component to MessageDetailsScreen

## Short description
This PR adds the new footer to `MessageDetailsScreen`. Currently, we have only two actions: _contact the sender_ and _show more data_

<details open><summary>Details</summary>
<p>

| footer |
| - |
| <video src="https://github.com/pagopa/io-app/assets/29163287/d88b9427-e2a7-4102-b2d8-c071a04a434d" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- update locales
- added `MessageDetailsFooter` component
- updated `MessageDetailsScreen` to display the `MessageDetailsFooter` component

## How to test
Using `io-dev-api-server`, generate a message. Activate experimental ds and open a message. Check that the footer is displayed correctly.
